ablog/post.py

```diff
@@ -741,10 +741,7 @@ def generate_atom_feeds(app):
 
             # Entry values that support templates
             title = post.title
-            if post.excerpt:
-                summary = " ".join(paragraph.astext() for paragraph in post.excerpt[0])
-            else:
-                summary = ""
+            summary = "".join(paragraph.astext() for paragraph in post.excerpt)
```

Comment:
So just to double check, if "post.excerpt" were to be empty, this would not error now?

Reply:
In theory, but in the commit message I note I don't have a test case for this.  The way I read the exception in #96, it was happening because `post.excerpt` is an empty list, in which case the generator expression should work just fine resulting in an empty summary string, as is appropriate.  LMK if you want me to add empty post coverage for this to the  `./tests/test_build.py::test_feed` test.

Reply:
Only if you want, otherwise I am happy to merge this PR and do a release.

Reply:
I'm going ahead and adding coverage for this.  Standby.

Reply:
I'm getting intermittent, seemingly unrelated failures from the whole `$ tox -p all` suite, though `$ ./.tox/py37-sphinxdev/bin/pytest` is passing for me.  So I decided to push and see what CI and you have to say about it.  LMK if you want me to revert.